app/ios/Runner.xcodeproj/project.xcworkspace/contents.xcworkspacedata

```diff
@@ -2,6 +2,6 @@
 <Workspace
    version = "1.0">
    <FileRef
-      location = "group:Runner.xcodeproj">
+      location = "self:">
```

Comment:
This seems suspicious.

Reply:
Ill try to change this back to group and see what happens.  I did not touch this file whatsoever, maybe during flutter upgrade